lib/subscribers/langchain/runnable-stream.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const LangchainSubscriber = require('./base')
+const { AI: { LANGCHAIN, STREAMING_DISABLED } } = require('../../metrics/names')
+
+class LangchainRunnableStreamSubscriber extends LangchainSubscriber {
+  constructor({ agent, logger }) {
+    super({ agent, logger, channelName: 'nr_stream' })
+    this.events = ['asyncEnd']
+  }
+
+  get enabled() {
+    return super.enabled && this?.agent?.config?.ai_monitoring?.streaming?.enabled
+  }
+
+  handler(data, ctx) {
+    const { agent, logger } = this
+    // We need these checks inside the handler because it is possible for monitoring
+    // to be disabled at the account level. In such a case, the value is set
+    // after the instrumentation has been initialized.
+    if (!agent?.config?.ai_monitoring?.enabled) {
+      this.logger.debug('Langchain instrumentation is disabled via `config.ai_monitoring.enabled`, not creating segment.')
+      return ctx
+    }
+    if (!agent.config?.ai_monitoring?.streaming?.enabled) {
```

Comment:
No need to do optional chaining

```suggestion
    if (!agent.config.ai_monitoring.streaming.enabled) {
```